cityofaustin/atd-moped
# Add funding source column to project list view

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9006

Adds column as well as option to filter on column. 

I also removed `project_importance`, `project_length` and `project_priority` from the SQL View since they are not needed for the Projects List View page.

## Testing
**URL to test:** 

moped test or test locally. 

Note -- ~~I branched this branch (https://github.com/cityofaustin/atd-moped/pull/642) off of this branch, so if you deploy PR #642 to test, you will be able to test both of these enhancements in one deployment~~ You can now deploy the branch from https://github.com/cityofaustin/atd-moped/pull/645 if you want to test the funding source column, the status updat column and the project type bug fix in one build.  cc: @amenity 

**Steps to test:**

Add multiple funding sources to a project https://moped-test.austinmobility.io/moped/projects/174?tab=funding

Go back to https://moped-test.austinmobility.io/moped/projects/, toggle the column on and see the sources. 

---
#### Ship list
